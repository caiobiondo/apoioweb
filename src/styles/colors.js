//
// Color Specs
// -------------------------------------
export const black = '#000';
export const white = '#fff';

export const gray100 = '#eeeeee';
export const gray150 = '#dddddd';
export const gray200 = '#bbbbbb';
export const gray300 = '#8e8e8e';
export const gray600 = '#606060';
export const gray890 = '#333333';
export const gray900 = '#222222';

export const orange100 = '#f4a652';
