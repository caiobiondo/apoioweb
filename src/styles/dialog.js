export const dialogContainer = {
  maxWidth: 'none',
};

export const dialogContent = {
  fontFamily: 'Roboto',
  padding: '12px 25px',
};

export const dialogTitle = {
  fontWeight: 800,
};

export const dialogActions = {
  fontFamily: 'Roboto',
};
