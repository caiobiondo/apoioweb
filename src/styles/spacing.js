//
// Spacing
// -------------------------------------
export const spSmall = '5px';
export const spMedium = '24px';
export const spPage = '40px';

export const screenXs = '576px';
export const screenSm = '576px';
export const screenMd = '768px';
export const screenLg = '992px';
export const screenXl = '1200px';
