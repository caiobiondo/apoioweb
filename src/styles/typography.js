//
// Font Weights
// -------------------------------------
export const fw300 = '300';
export const fw400 = '400';
export const fw600 = '600';
export const fw700 = '700';
export const fw900 = '900';

//
// Font Sizes
// -------------------------------------
export const fs9 = '9px';
export const fs10 = '10px';
export const fs12 = '12px';
export const fs13 = '13px';
export const fs14 = '14px';
export const fs15 = '15px';
export const fs16 = '16px';
export const fs18 = '18px';
export const fs20 = '20px';
export const fs25 = '25px';
export const fs28 = '28px';
export const fs32 = '32px';

//
// Font Families
//

export const NaturaBold = 'NaturaSans-Bold';
export const NaturaRegular = 'NaturaSans-Regular';
export const RobotoRegular = 'Roboto-Regular';
export const RobotoMedium = 'Roboto-Medium';
export const RobotoLight = 'Roboto-Light';
