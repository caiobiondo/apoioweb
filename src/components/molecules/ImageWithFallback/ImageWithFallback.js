import React from 'react';
import Img from 'react-image';
import { CircularProgress, Icon } from 'natura-ui';
import { Wrapper } from './ImageWithFallback.styles';

const ImageWithFallback = props => {
  const fallbackImage = props.unloader ? props.unloader : <Icon file={props.fallbackIcon} />;
  const loader = <CircularProgress thickness={2} />;

  return (
    <Wrapper width={props.width} height={props.height || props.width} className={props.className}>
      <Img src={props.imageUrl} loader={loader} unloader={fallbackImage} />
    </Wrapper>
  );
};

ImageWithFallback.defaultProps = {
  fallbackIcon: 'ico_pictureless',
};

export default ImageWithFallback;
