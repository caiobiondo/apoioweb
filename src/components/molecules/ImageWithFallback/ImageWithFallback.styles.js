import styled from 'styled-components';
import { gray200 } from 'styles/colors';

export const Wrapper = styled.div`
  svg,
  img {
    width: ${props => props.width};
    height: ${props => props.height};
  }

  svg {
    fill: ${gray200};
  }
`;
