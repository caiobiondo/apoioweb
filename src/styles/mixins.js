import styled, { css } from 'styled-components';
import { Responsive } from '@entria/components';
import { spMedium } from 'styles/spacing';

export const Full = css`
  position: absolute;
  top: 0;
  left: 0;
  right: 0;
  bottom: 0;

  @media (max-width: ${Responsive.VIEWPORT.MEDIUM}px) {
    top: 0;
    left: ${spMedium};
    right: ${spMedium};
    bottom: 0;
  }
`;

export const Center = css`
  display: flex;
  flex-direction: column;
  justify-content: center;
  align-items: center;
  height: 100%;
`;

export const LoadingWrapperStyle = styled.div`
  display: flex;
  flex-direction: column;
  justify-content: center;
  align-items: center;
  height: 100%;
  margin-top: 40px;
`;
