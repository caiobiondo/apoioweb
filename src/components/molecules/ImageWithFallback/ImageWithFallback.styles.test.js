import React from 'react';
import { shallow } from 'enzyme';
import { Wrapper } from './ImageWithFallback.styles';

describe('ImageWithFallback Styles', () => {
  it('should render correctly', () => {
    const result = shallow(<Wrapper height="10px" width="10px" />);
    expect(result).toMatchSnapshot();
  });
});
