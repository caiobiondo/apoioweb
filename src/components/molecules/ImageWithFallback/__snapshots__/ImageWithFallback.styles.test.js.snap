// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ImageWithFallback Styles should render correctly 1`] = `
<div
  className="ImageWithFallbackstyles__Wrapper-kVaXrS cxoRrX"
  height="10px"
  width="10px"
/>
`;
